translations/ja/ja_test.go

```diff
@@ -139,6 +139,7 @@ func TestTranslations(t *testing.T) {
 		StrPtrGte         *string   `validate:"gte=10"`
 		OneOfString       string    `validate:"oneof=red green"`
 		OneOfInt          int       `validate:"oneof=5 63"`
+		Image			  string			`validate:"image"`
```

Comment:
type and tag and tag not aligned with lines above

Reply:
Thank you. I'll work on that.